server.go

```diff
@@ -75,7 +80,7 @@ func NewServer(opts ...Option) *Server {
 	}
 
 	// update struct...perhaps there is a better way.
-	s.Addr = fmt.Sprintf("127.0.0.1:%d", s.Port)
+	s.Addr = fmt.Sprintf(":%d", s.Port)
```

Comment:
We conventionally use [`net.JoinHostPort`](https://pkg.go.dev/net#JoinHostPort) for this, because a colon doesn't work with IPv6 addresses, for example.